apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php

```diff
@@ -24,6 +24,11 @@ class Help_Center {
 	public function __construct() {
 		add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_script' ), 100 );
 		add_action( 'rest_api_init', array( $this, 'register_rest_api' ) );
+		// Crazy high number inorder to prevent Jetpack removing it
+		// https://github.com/Automattic/jetpack/blob/30213ee594cd06ca27199f73b2658236fda24622/projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php#L196.
+		add_action( 'wp_before_admin_bar_render', array( $this, 'admin_bar_menu' ), 100000 );
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_script' ), 100 );
```

Comment:
This is causing an error in the block editor because `enqueue_script` is called twice (once here and once during `enqueue_block_editor_assets`), which adds an inline script to define a global called `helpCenterFeatureFlags`. I think it's behind a feature flag at the moment, so probably doesn't cause many problems.

<img width="720" alt="image" src="https://user-images.githubusercontent.com/96462/194949434-eb0d694d-8356-4bad-adde-fe48aa38b6c7.png">